### ISSUE TITLE
check and fix translation and validation messages

### ISSUE THREAD

**Issue by boris-brtan** [shopsys/shopsys#884]
### What is happening

i found out that there are some translation bugs based on english language where instead of word `than` there is used `then` and also there is some strange form of mail word like capitalized it is without dash but not capitalized with dash like `Email cannot be longer then` and `Please enter e-mail`
```
First name cannot be longer then {{ limit }} characters
Email cannot be longer then {{ limit }} characters
Password cannot be longer then {{ limit }} characters

```

### Expected result

have it fixed and maybe some checker for *.po files

**Comment by LukasHeinz** [comment]
I see two issues here:
1) Do not use in English words with dash `e-mail` and `e-commerce`.
2) Check and fix the usage of `then` and `than` 

I do not recommend to use automatic check or fixer, because for some special reasons we might to use a different form...

**Comment by s3tezsky** [comment]
fixed in #1335
